### PR TITLE
Install kubectx on management node #457

### LINKED ISSUE #457: Proposal: Automatic Installation of kubectx for Enhanced Cluster Management
As a user of the management server, I propose the addition of automatic installation of the tool kubectx. This enhancement will improve the cluster management user experience and provide a more streamlined experience for users interacting with multiple Kubernetes clusters.

Kubectx is a popular command-line utility that simplifies switching between multiple Kubernetes contexts, namespaces and clusters. By integrating kubectx into the management server's setup process, we can enable users to manage and navigate different clusters more efficiently.

However, this becomes more useful only when managing larger installations.

These tools are often used with aliases `kctx` and `kns`.

https://github.com/ahmetb/kubectx

kubectx demo

kubens demo

With the installation of an additional plugin, it enables interactive behaviour and search.

fuzzy interactive search

### PR DESCRIPTION
**Pull request by fdobrovolny**
Closes #457

### CODE REVIEW

File: terraform/files/bin/install_kubectx.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+sudo apt-get install --no-install-recommends --no-install-suggests -y fzf
+
+# Deploy kubectx on mgmt cluster for convenience
+git clone https://github.com/ahmetb/kubectx.git ~/.kubectx
```
Comment (matofeder) [inline terraform/files/bin/install_kubectx.sh:6]: Minor: pin a version and provide a short log msg, e.g.:
```
KUBECTX_VERSION=<TAG>
echo "# install kubectx $KUBECTX_VERSION"
git clone --depth 1 --branch <TAG> https://github.com/ahmetb/kubectx.git ~/.kubectx
```

File: terraform/files/bin/bootstrap.sh [hunk 2]
```diff
@@ -87,6 +87,7 @@ install_helm.sh
 deploy_cluster_api.sh
 install_k9s.sh
 get_capi_helm.sh
+install_kubectx.sh
```
Comment (matofeder) [inline terraform/files/bin/bootstrap.sh:90]: Minor: Would be great (sooner is better) to introduce the `Release-Notes-R6.md` doc and fill it appropriately.

**Review by matofeder — APPROVED** [review]
Lgtm

**Comment by fdobrovolny** [comment]
@matofeder addressed ready for merge, will do a separate PR to update the release notes

**Pull request merged**